config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-ci.yaml

```diff
@@ -41,3 +41,22 @@ periodics:
   annotations:
     testgrid-dashboards: sig-api-machinery-structured-merge-diff
     testgrid-tab-name: ci-gofmt
+- name: ci-structured-merge-diff-benchmark
+  interval: 12h
+  labels:
+    preset-service-account: "true"
+  decorate: true
+  spec:
+    containers:
+    - image: golang:latest
+      command:
+      - go
+      args:
+      - run
+      - k8s.io/test-infra/pkg/benchmarkjunit
```

Comment:
I don't know if this will work. I couldn't get `go run` working with go modules so we've had to check out the test-infra repo with `extra_refs` and reference by path: https://github.com/kubernetes/test-infra/blob/4225c594bbbf3082ce5c3a861d556ec3a7eda431/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml#L451-L481

If you know how to make `go run` work with go modules that would be nice to get working, but I ultimately gave up.

Reply:
Ah right.

To address this, I've tried building a new image w/ both the benchmarkjunit binary and the go toolchain. PTAL at the change + my test.